tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/InstrumentationTests.cs

```diff
@@ -492,7 +492,7 @@ public async Task OnSupportedFrameworkInSsi_CallsForwarderWithExpectedTelemetry(
                                "tags": ["injection_forced:false"]
                              }]
                              """;
-            AssertHasExpectedTelemetry(logFileName, processResult, pointsJson);
+            AssertHasExpectedTelemetry(logFileName, processResult, pointsJson, "success", "Successfully configured ddtrace package", "success");
```

Comment:
If we update the description above we need to remember to update these!

```suggestion
            AssertHasExpectedTelemetry(logFileName, processResult, pointsJson, "success", "Successfully configured automatic instrumentation", "success");
```